PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,46 @@
-Type: patch
+### PR Titles
+#### To pass testing pipeline, these need to follow Conventional Commits spec:
+https://www.conventionalcommits.org/en/v1.0.0/
+<br>
+e.g.
+<br>
+`feat: create NewForm component.`
+<br>
+or
+<br>
+`fix: update broken link in NewForm component.`
+<br>
+<br>
 
-Fixes #
 
-## Changes proposed in this PR
+### PR description
+#### What is it doing?
+Tell us what it is doing from a technical perspective.
 
-- Briefly describe the changes in this PR
-- Do you have some test asserting this behavior?
+#### Why is this required?
+Tell us why this is required from a business/product perspective.
+
+#### link to Jira ticket:
+Add a link to the Jira ticket.
+<br>
+<br>
+
+
+### Quick Checklist:
+<input type="checkbox">
+<label>My PR title follows the Conventional Commit spec.</label>
+<br>
+
+<input type="checkbox">
+<label>I have filled out the PR description as per the template above.</label>
+<br>
+
+<input type="checkbox">
+<label>I have added tests to cover new or changed behaviour.</label>
+<br>
+
+<input type="checkbox">
+<label>I have updated any relevant documentation.</label>
+<br>
```

Comment:
GitHub markdown supports checklists natively 🙂 

```markdown
- [ ] this is a tickbox
- [ ] here's another one
```

Reply:
Odd, I'm 90% sure that I was able to click them in the past... Just to test I ticked a couple in my earlier comment and it seems to work? Possible that it's only allowed if you're the person who wrote the comment in the first place. It also wouldn't work in the Markdown file preview since this is not editable.